### PR TITLE
Added basic functionality for Uninstall-BicepCLI

### PR DESCRIPTION
**Pull request by bjompen**
This version does not force clean the $env:path variable.

### CODE REVIEW

**Comment by bjompen** [comment]
forgot to tag issue...
https://github.com/StefanIvemo/BicepPowerShell/issues/11

File: Source/Public/Uninstall-BicepCLI.ps1 [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            }
+            else {
+                # Bicep is running in standalone exe mode. Remove manualy
+                $ExePath = Join-path -Path $Install -ChildPath $BicepExeName
+                if (Test-Path $ExePath) {
+                    Remove-Item $ExePath
+                }
+            }
+        }
+
```
Comment (StefanIvemo) [inline Source/Public/Uninstall-BicepCLI.ps1:37]: The test to verify if uninstall was successful or not runs to quick after the uninstaller is finished. The terminal session still has bicep loaded as a command for a couple of seconds. Got it to work as expected by adding `Start-Sleep 5` before the validation.

**Review by StefanIvemo — CHANGES_REQUESTED** [review]
Looks good! Take a look at the comments. I do get an error when uninstalling the windows installer version of Bicep CLI.

File: Source/Public/Uninstall-BicepCLI.ps1 (hunk 1 above)
Comment (bjompen) [reply Source/Public/Uninstall-BicepCLI.ps1:41]: fixed in commit https://github.com/StefanIvemo/BicepPowerShell/pull/14/commits/fb98e1b83fbee83f0696a29628722617e5e10edb

**Comment by StefanIvemo** [comment]
@bjompen Thank you for another great contribution! ❤

**Pull request merged**